### PR TITLE
COL-828 Add 'impact' and 'trending' columns to assets

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-828

### CODE REVIEW

File: node_modules/col-core/lib/db.js [hunk 1]
```diff
@@ -468,6 +468,22 @@ var setUpModel = function(sequelize) {
...
+      'defaultValue': 0
+    },
+    'trending_percentile': {
+      'type': Sequelize.INTEGER,
+      'defaultValue': 0
+    },
+    'trending_score': {
+      'type': Sequelize.INTEGER,
+      'defaultValue': 0
+    },
```
Comment (johncrossman) [inline node_modules/col-core/lib/db.js]: @paulkerschen no need for `'allowNull': false` above?

Comment (paulkerschen) [inline node_modules/col-core/lib/db.js]: @johncrossman Good question; existing code is inconsistent on that point. But I've amended the new columns to make it explicit.

File: scripts/20170426-col828/add_impact_trending_to_assets.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+-- Add per-asset columns containing 'impact' and 'trending' scores and percentiles. These values
+-- are periodically recalculated and used interally for query selection and ordering, but do not 
+-- surface through the API.
+
+ALTER TABLE assets ADD impact_percentile integer NOT NULL DEFAULT 0;
+ALTER TABLE assets ADD impact_score integer NOT NULL DEFAULT 0;
+ALTER TABLE assets ADD trending_percentile integer NOT NULL DEFAULT 0;
+ALTER TABLE assets ADD trending_score integer NOT NULL DEFAULT 0;
```
Comment (johncrossman) [inline scripts/20170426-col828/add_impact_trending_to_assets.sql]: @paulkerschen Fyi, I believe the four ADDs above can live in a single ALTER statement:
```
ALTER TABLE table ADD COLUMN col1 int, ADD COLUMN col2 int, ...;
```
Gives you benefit of all or nothing. But, this is a one time script so I don't see a need for change.

**Comment by johncrossman** [comment]
Waiting on T...

**Pull request merged**